Assets/Mirror/Runtime/Transport/Websocket/ClientJs.cs

```diff
@@ -40,9 +40,12 @@ public bool IsConnected
         int m_NativeRef = 0;
         readonly int id;
 
-        public Client()
+        static byte[] buffer;
+
+        public Client(int BufferSize)
         {
             id = Interlocked.Increment(ref idGenerator);
+            buffer = buffer ?? new byte[BufferSize];
```

Comment:
what if someone does:
new Client(100);
new Client(200);
?
in this case the second client's buffer size would be 100 instead of 200.
didn't you just create it every time before? should be fine considering how there is usually only one client at a time anyway